src/components/ui/CustomModal.tsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from "react";
+import CustomButton from "./CustomButton";
+
+interface ModalProps {
+  openModal: boolean;
+}
+
+const CustomModal = ({ openModal }: ModalProps) => {
+  const [ clicked, setClicked ] = useState(false)
+
+  return (
+    <div
+      className={`${
+        openModal ? "block" : "hidden"
+      } " fixed top-0 left-0 h-screen w-full bg-slate-800 bg-opacity-60 "`}
```

Comment:
I feel your modal should also have like a prop for "maxWidth" , just incase one wants to change the width of the modal